nwb_conversion_tools/NWBConverter.py

```diff
@@ -0,0 +1,58 @@
+
+from copy import deepcopy
+from utils import get_schema_from_hdmf_class
+from pynwb.file import NWBFile, Subject
+from BaseDataInterface import NeuroscopeRecordingInterface
+
+base_schema = dict(
+    required=[],
+    properties={},
+    type='object',
+    additionalProperties='false')
+
+root_schema = deepcopy(base_schema)
+root_schema.update({
+    "$schema": "http://json-schema.org/draft-07/schema#",
+})
```

Comment:
can we just define these once? I see them also in BaseDataInterface

Reply:
Would it work for you if I made a separate function in utils called `make_root_schema` or something like that and had each of them call it?

Reply:
sure

Reply:
Done!